lib/model/src/main/resources/META-INF/services/io.atlasmap.v2.Action

```diff
@@ -0,0 +1,61 @@
+io.atlasmap.v2.AbsoluteValue
+io.atlasmap.v2.Add
+io.atlasmap.v2.AddDays
+io.atlasmap.v2.AddSeconds
+io.atlasmap.v2.Append
```

Comment:
Custom actions models can now just extend Action and list that class in their own 'META-INF/services/io.atlasmap.v2.Action' file.  Service discovery should add it and Jackson should be able to deserialize it.